products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,233 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - this tutorial could contain Game of Thrones ***spoilers***.  
+
+Game of Thrones is an American fantasy drama television series created by David Benioff and D. B. Weiss for HBO.
+It is an adaptation of A Song of Ice and Fire, George R. R. Martin's series of fantasy novels,  the first of which is
+A Game of Thrones. The Game of Thrones world is full of interesting characters, locations, scenarios, unexpected deaths, and plot twists.
+
+Even though the COVID-19 pandemic hit the entire world in 2020 and is now starting to become one of the worst
+years in recent history, 2019 was also a huge disappointment to all the Game of Thrones fans. According to user reactions, a seven-year 
+build-up resulted in a poorly written ending of the last season and ruined the ending of one of the most popular shows on the planet.  
+Nonetheless, if you want to know how many characters would have survived if Jon Snow had stayed dead, which House had the best Kill/Death Ratio,
+or who was the biggest traitor in the show, you came to the right place!
+
+### Data Model
+
+Although the Game of Thrones TV show is based on a series of books, our graph database contains only characters from 
+previously mentioned TV shows as the books are still not finished. This tutorial would not be possible with data analyst David 
+Murphy who shared his collection of on-screen deaths on [this link](https://data.world/datasaurusrex/game-of-thones-deaths). We won't be working with kills and deaths that happened off-screen or were 
+tied to the undead (wraiths). The dataset we used was slightly modified, columns "Episode Name" and "IMDb Rating" were added.

```

Comment:
_(optional) opinion:_ This sentence is a bit clumsy. I think it would sound better if you replaced the comma with some sort of conjunction or split the sentences with a full-stop.